CITATION.cff

```diff
@@ -0,0 +1,124 @@
+cff-version: 1.2.0
+message: "If you use this book, please cite it as below."
+title: "SnowEx Hackweek JupyterBook Tutorials"
```

Comment:
note this is a single citation for the entire book/website, rather than broken down per tutorial.